financial-connections/src/main/java/com/stripe/android/financialconnections/model/FinancialConnectionsSession.kt

```diff
@@ -44,7 +47,11 @@ data class FinancialConnectionsSession internal constructor(
     @SerialName("bank_account_token")
     @Serializable(with = JsonAsStringSerializer::class)
     internal val bankAccountToken: String? = null
-) : StripeModel, Parcelable
+) : StripeModel, Parcelable {
+
+    val accounts: FinancialConnectionsAccountList
+        get() = _accountsOld ?: _accountsNew!!
```

Comment:
I think it should be the other way around, aka we favor _accountsNew over _accounts old

Reply:
I'll change it! but this assumes there's always one and just one one of them available, so it should work either way : )